### PR TITLE
Fix #873

### LINKED ISSUE #873: SyntaxError: Unexpected token
<!--
IF YOU ARE POSTING A QUESTION HOW TO USE MINEFLAYER,
PLEASE ASK ON OUR DISCORD SERVER.

https://discord.gg/eybmzRR
-->

## Versions
 - mineflayer: 2.9.6
 - server: spigot 1.12.2
 - node: 8.11.4

## Detailed description of a problem
I tried to connect to a server with the bot, but the bot don't work, I don't really know why.

## Current code
```js
var mineflayer = require('mineflayer');
var bot = mineflayer.createBot({
  host: "play.hadaria.fr", // optional
  port: 25565,       // optional
  username: "MyEmail", // email and password are required only for
  password: "MyPassword",          // online-mode=true servers
  version: "1.12.2",               // false corresponds to auto version detection (that's the default), put for example "1.8.8" if you need a specific version
  keepAlive: true,
 });
bot.on('message', (message) => {
	console.log(message.toAnsi());
});
bot.on('error', err => console.log(err))
```

## Expected behavior
I've tried to read the chat in the cmd

## Additional context
I have this error when I try to join the server :
```C:\Users\Admin\Desktop>node bot1.js
Auto logged in
Connexion » Connexion effectuée !

undefined:1
{"text":"","extra":[{"text":"\n","color":"yellow"},{"text":"✦ ","color":"yellow"},{"text":"HADARIA ","color":"gold","bold":"true"},{"text":"• ","color":"gray"},{"text":"PvP/Factions ","color":"gold","bold":"true"},{"text":"Farm2Win ","color":"green","bold":"true"},{"text":"N","color":"red","bold":"true"},{"text":"°","color":"red"},{"text":"1 FR ","color":"red","bold":"true"},{"text":"✦
                                                                                                                                                                                                                                                                                                                                                                                                    ^

SyntaxError: Unexpected token
 in JSON at position 388
    at JSON.parse (<anonymous>)
    at Client.bot._client.on (C:\Users\Admin\node_modules\mineflayer\lib\plugins\tablist.js:18:49)
    at emitTwo (events.js:126:13)
    at Client.emit (events.js:214:7)
    at FullPacketParser.deserializer.on (C:\Users\Admin\node_modules\minecraft-protocol\src\client.js:82:12)
    at emitOne (events.js:116:13)
    at FullPacketParser.emit (events.js:211:7)
    at addChunk (C:\Users\Admin\node_modules\readable-stream\lib\_stream_readable.js:297:12)
    at readableAddChunk (C:\Users\Admin\node_modules\readable-stream\lib\_stream_readable.js:279:11)
    at FullPacketParser.Readable.push (C:\Users\Admin\node_modules\readable-stream\lib\_stream_readable.js:240:10)```

### PR DESCRIPTION
**Pull request by wvffle**
I'm matching whole replaced string so I don't have to add `": "` later.
Also I'm replacing every matched string which fixes #873

### CODE REVIEW

File: test/internalTest.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ mineflayer.supportedVersions.forEach((supportedVersion, i) => {
           username: 'player',
           version: supportedVersion,
           port: 25567
+          // logErrors: false
```
Comment (rom1504) [inline test/internalTest.js]: Why ?

File: lib/plugins/tablist.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module.exports = inject
 
 const escapeValueNewlines = str => {
-  return str.replace(/": *"((?:\\"|[^"])+)/, (_, match) => '":"' + match.replace('\n', '\\n'))
+  return str.replace(/(": *"(?:\\"|[^"])+")/g, (_, match) => match.replace('\n', '\\n'))
```
Comment (rom1504) [inline lib/plugins/tablist.js]: Why ? What's the purpose ? Please explain in PR description

Comment (wvffle) [reply lib/plugins/tablist.js]: I'm matching whole replaced string so I don't have to add `": "` later. Also I'm replacing every matched string which fixes #873

File: test/internalTest.js (hunk 1 above)
Comment (wvffle) [reply test/internalTest.js]: I was debugging why my test does not work, going to remove it quickly

**Pull request merged**